pkg/arvo/app/invite-store.hoon

```diff
@@ -1,184 +1,206 @@
 ::  invite-store [landscape]
-/+  *invite-json, default-agent, dbug
+/-  store=invite-store
+/+  res=resource, default-agent, dbug
 |%
 +$  card  card:agent:gall
-::
 +$  versioned-state
-  $%  state-zero
+  $%  state-0
+      state-1
   ==
 ::
-+$  state-zero
-  $:  %0
-      =invites
++$  invitatory-0  (map serial:store invite-0)
++$  invite-0
+  $:  =ship           ::  ship to subscribe to upon accepting invite
+      app=@tas        ::  app to subscribe to upon accepting invite
+      =path           ::  path to subscribe to upon accepting invite
+      recipient=ship  ::  recipient to receive invite
+      text=cord       ::  text to describe the invite
   ==
+::
++$  state-0  [%0 invites=(map path invitatory-0)]
++$  state-1  [%1 =invites:store]
 --
 ::
-=|  state-zero
+=|  state-1
 =*  state  -
 %-  agent:dbug
 ^-  agent:gall
-=<
-  |_  bol=bowl:gall
-  +*  this      .
-      inv-core  +>
-      ic        ~(. inv-core bol)
-      def       ~(. (default-agent this %|) bol)
-  ++  on-init   on-init:def
-  ++  on-save   !>(state)
-  ++  on-load
-    |=  old=vase
-    `this(state !<(state-zero old))
+::
+|_  =bowl:gall
++*  this  .
+    def   ~(. (default-agent this %|) bowl)
+::
+++  on-init
+  ^-  (quip card _this)
+  :-  ~
+  %_  this
+      invites.state
+    %-  ~(gas by *invites:store)
+    [%graph *invitatory:store]~
+  ==
+::
+++  on-save   !>(state)
+++  on-load
+  |=  old-vase=vase
+  =/  old  !<(versioned-state old-vase)
+  ?:  ?=(%1 -.old)
+   `this(state old)
+  :-  =-  [%pass / %agent [our.bowl %invite-store] %poke %invite-action !>(-)]~
+      !>  ^-  action:store
+      [%create %graph]
+  %=  this
+      state
+    :-  %1
+    %-  ~(gas by *invites:store)
+    %+  murn  ~(tap by invites.old)
+    |=  [=path =invitatory-0]
+    ^-  (unit [term invitatory:store])
+    ?.  ?=([@ ~] path)  ~
+    :-  ~
+    :-  i.path
+    %-  ~(gas by *invitatory:store)
+    %+  turn  ~(tap by invitatory-0)
+    |=  [=serial:store =invite-0]
+    ^-  [serial:store invite:store]
+    :-  serial
+    ^-  invite:store
+    :*  ship.invite-0
+        app.invite-0
+        (de-path:res path.invite-0)
+        recipient.invite-0
+        text.invite-0
+    ==
+  ==
+::
+++  on-agent  on-agent:def
+++  on-arvo   on-arvo:def
+++  on-leave  on-leave:def
+++  on-fail   on-fail:def
+::
+++  on-watch
+  |=  =path
+  ^-  (quip card _this)
+  ?>  (team:title our.bowl src.bowl)
+  =/  cards=(list card)
+    ?+  path  (on-watch:def path)
```

Comment:
`path` should be indented once more, or the cases (`[%all ~]`, etc.) should be dedented.